release/src/stdlib_dependabot/file_update/main.py

```diff
@@ -0,0 +1,226 @@
+import urllib.request
+import json
+import sys
+from retry import retry
+import os
+import semver
+from github import Github, InputGitAuthor, GithubException
+
+HTTP_REQUEST_RETRIES = 3
+HTTP_REQUEST_DELAY_IN_SECONDS = 2
+HTTP_REQUEST_DELAY_MULTIPLIER = 2
+packageUser =  os.environ["packageUser"]
+packagePAT = os.environ["packagePAT"]
+packageEmail =  os.environ["packageEmail"]
+organization = 'ballerina-platform'
+
+def main():
+    modulesWithVersionUpdates = preprocessString()
+    stdlibModules = getModuleListFromFile()
+    updatedModuleDetails = initializeModuleDetails(modulesWithVersionUpdates)
+    updatedModuleDetails = getImmediateDependents(stdlibModules, updatedModuleDetails)
+    updateFiles(updatedModuleDetails)
+
+# Fetch the updated module versions string from the environment and append it to an array
+def preprocessString():
+    try:
+        # String is in the form -> '"module": "version" "module": "version"'
+        updatedModules = os.environ["modules"]
+        updatedModules = updatedModules.split()
+
+        latestVersion = []
+        for i in range(0, len(updatedModules)-1, 2):
+            latestVersion.append(updatedModules[i][1:-2] + ':' + updatedModules[i+1][1:-1]) 
+
+        # Processed string -> ["module:version", ",module:version"]
+        return latestVersion
+    except:
+        print("Failed to update files")
+        print("Input String: " + os.environ["modules"])
+        sys.exit()
+
+# Fetch list of stdlib modules from stdlib_latest_versions.json 
+def getModuleListFromFile():
+    try:
+        with open('./release/resources/stdlib_latest_versions.json') as f:
+            moduleList = json.load(f)
+    except:
+        print('Failed to read stdlib_latest_versions.json')
+        sys.exit()
```

Comment:
Shall we add an exit code here? This must be `!=0` right?

Check the other places as well.

Reply:
Updated